docs/source/example_notebooks/dowhy_simple_example.ipynb

```diff
@@ -481,11 +511,105 @@
    "source": [
     "**Conclusion**: Assuming that the unobserved confounder does not affect the treatment or outcome more strongly than any observed confounder, the causal effect can be concluded to be positive."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Running Refutation Methods in Parallel\n",
+    "\n",
+    "You can spread the workload to multiple CPUs to speed up the refutation process.\n",
+    "\n",
+    "**Method 1**: Parallelize a single refutation method using the `n_jobs` parameter. For example, set `n_jobs=2` to spread the workload to two CPUs, or set `n_jobs=-1` to use all available cores."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "\n",
+    "res_placebo=model.refute_estimate(identified_estimand, estimate,\n",
+    "        method_name=\"placebo_treatment_refuter\", placebo_type=\"permute\", n_jobs=-1, verbose=3)\n",
+    "print(res_placebo)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Method 2**: Similarly, you can run multiple refutation methods in parallel using the code below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from joblib import Parallel, delayed\n",
+    "from functools import partial\n",
+    "\n",
+    "# put kwargs in dictionaries and make a list\n",
+    "kwargs_common = dict(\n",
+    "    method_name=\"random_common_cause\")\n",
+    "\n",
+    "kwargs_random = dict(\n",
+    "    method_name=\"placebo_treatment_refuter\",\n",
+    "    placebo_type=\"permute\")\n",
+    "\n",
+    "kwargs_subset = dict(\n",
+    "    method_name=\"data_subset_refuter\",\n",
+    "    subset_fraction=0.9,\n",
+    "    random_seed=1)\n",
+    "\n",
+    "\n",
+    "kwargs_list = [kwargs_common, kwargs_random, kwargs_subset]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "\n",
+    "# partial function to fix some arguments\n",
+    "refute = partial(model.refute_estimate, identified_estimand, estimate)\n",
+    "\n",
+    "# run refutation methods in parallel\n",
+    "results = Parallel(n_jobs=4, verbose=10)(delayed(refute)(**kwds) for kwds in kwargs_list)\n",
```

Comment:
as you mentioned, this code can now be removed from the notebook and you can add a description for the n_jobs parameter when the inidividual methods are called.

Reply:
Ah I forgot to remove the section, will do with the next commit.